draft-ietf-tsvwg-rfc4960-bis.xml

```diff
@@ -1288,8 +1289,8 @@ The value 0 MUST NOT be used.</t>
 two endpoints will use the min(requested, offered).
 See <xref target='sec_handle_stream_parameters'/> for details.</t>
 <t>A receiver of an INIT chunk with the MIS value set to 0 MUST discard the
-packet, SHOULD send a packet in response containing an ABORT chunk and using
-the Initiate Tag as the Verification Tag, and MUST NOT change the state of any
+packet, SHOULD send a packet in response containing an ABORT chunk with the
+T bit set, and MUST NOT change the state of any
```

Comment:
And which Verification Tag should be used?
* If you use 0, the receiver can not verify it. This also violates the rule that only packets containing the an INIT chunk use a Verification Tag of 0.
* If you use the Initiate Tag, the receiver will drop the packet containing the ABORT chunk.

So are you suggesting that no packet containing an ABORT chunk is used, since it does not make a lot of sense to send a packet knowing that the receiver will drop it. It only makes sense for monitoring devices.

Reply:
Since I think the original text is better than the suggested, I'm resolving this conversation without changing any text. If you don't agree, please unresolve the conversation.